contracts/jobs/JobsManager.sol

```diff
@@ -175,11 +169,10 @@ contract JobsManager is IJobsManager, Verifiable, Manager {
         job.streamId = _streamId;
         job.transcodingOptions = _transcodingOptions;
         job.maxPricePerSegment = _maxPricePerSegment;
-        job.pricePerSegment = pricePerSegment;
         job.broadcasterAddress = msg.sender;
         job.transcoderAddress = electedTranscoder;
 
-        NewJob(electedTranscoder, msg.sender, numJobs);
+        NewJob(electedTranscoder, msg.sender, numJobs, _streamId, _transcodingOptions);
```

Comment:
We currently cannot return streamId or transcodingOptions when using a proxy contract because the delegatecall mechanism cannot return dynamic length types (i.e. strings). Instead we can include these fields in events and a node could retrieve the values from the event